packages/coinstac-ui/app/render/utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+import { indexOf } from 'lodash';
+
+export const isPipelineOwner = (permissions, owningConsortium) => {
+  return indexOf(permissions.consortia[owningConsortium], 'owner') !== -1
```

Comment:
The old if's were looking into the `write` property and you changed to the `owner` property. Better take a look at this and make sure it's right.

Reply:
@rochaeb , I already checked it and it works.

Currently, permissions.consortia[owningConsortium] returns array like this - ['member', 'owner'] and using *.write does not make sense.

Please check this on your side with logging.

Reply:
Fixed with PR #731